FSI/Displacement.C

```diff
@@ -23,7 +23,7 @@ void preciceAdapter::FSI::Displacement::initialize()
 {
     // Initialize appropriate objects for each interface patch, namely the volField and the interpolation object
     // this is only necessary for face based FSI
-    if (this->locationsType_ == "faceCenters" || this->locationsType_ == "faceCentres")
+    if (this->locationsType_ == LocationType::faceCenters)
```

Comment:
[This page](https://precice.org/adapter-openfoam-config.html#the-adapters-configuration-file) will need an update to remove the `faceCentres` :gb: 

(original motivation: OpenFOAM uses British English for all configuration and this could have been an unexpected issue with some users. But better simplify)

Reply:
No, it won't, both spellings still work. I just summarized them into a single enum option.